Docker

```diff
@@ -0,0 +1,23 @@
+# Use an official Node.js runtime as the base image
+FROM node:14
+
+# Set the working directory inside the container
+WORKDIR /app
+
+# Copy the package.json and package-lock.json files to the container
+COPY package*.json ./
+
+# Install project dependencies
+RUN npm install --global @gridsome/cli
```

Comment:
Please reorder `RUN npm install --global @gridsome/cli` to be before `COPY package*.json ./`. The latter is expected to require execution more often than the gridsome CLI installation